src/pages/Policy.jsx

```diff
@@ -1,9 +1,160 @@
+              debitis pariatur, odit quidem, ipsam impedit vero aut corporis
+              accusantium quas, sed voluptates. Lorem ipsum dolor sit amet,
+              consectetur adipisicing elit. Nihil, qui. Alias sapiente nobis
+              soluta quo expedita fuga, atque eum. Sunt accusantium nobis non
+              minima voluptas itaque obcaecati rem! Sit itaque cupiditate,
+              nesciunt laborum perferendis quod corrupti quibusdam ad
+              repellendus optio esse ea. Ab, impedit ut. Animi modi corrupti non
+              dolorem?
+            </p>
+          </div>
+        </div>
+
+        <div className="privacy_policy">
+          <div>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Rerum
+              odit non ex excepturi, nihil tempore cum dignissimos odio,
+              consequuntur, reprehenderit tenetur. Fugit dicta perspiciatis
+              blanditiis ipsam libero unde cumque beatae asperiores alias
+              aperiam! Saepe perspiciatis corrupti, dolore sint aspernatur
+              veritatis rerum soluta assumenda quia, nesciunt quos, corporis
+              quibusdam tempora consequatur! Mollitia ab facere nostrum eaque
+              molestiae perferendis, doloremque commodi aliquid adipisci
+              aspernatur sed ut minima atque quibusdam quia, tenetur est eum
+              voluptatem? Libero aperiam quos facere aliquid, commodi corrupti
+              ex repellendus ipsam odio eveniet? Laborum provident rem veniam!
+              Itaque vitae quod, molestiae distinctio aliquid magnam iure,
+              sapiente quo omnis ex adipisci voluptatibus quisquam consequuntur
+              atque. Sapiente architecto tempora aperiam eaque tempore,
+              consectetur laboriosam atque consequatur obcaecati harum unde
+              itaque aliquam nam ducimus possimus magni quos earum cum fugit
+              corrupti velit corporis animi? Delectus eius velit et facere
+              debitis pariatur, odit quidem, ipsam impedit vero aut corporis
+              accusantium quas, sed voluptates. Lorem ipsum dolor sit amet,
+              consectetur adipisicing elit. Nihil, qui. Alias sapiente nobis
+              soluta quo expedita fuga, atque eum. Sunt accusantium nobis non
+              minima voluptas itaque obcaecati rem! Sit itaque cupiditate,
+              nesciunt laborum perferendis quod corrupti quibusdam ad
+              repellendus optio esse ea. Ab, impedit ut. Animi modi corrupti non
+              dolorem?
+            </p>
+          </div>
+          <div>
+            <img src={image4} alt="image1" />
+          </div>
+        </div>
+
+        <div className="privacy_policy">
+          <div>
+            <img src={image3} alt="image3" />
+          </div>
+          <div>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Rerum
```

Comment:
Store Lorem Ipsum text separately in constant folder and import wherever required